website/docs/docs/dbt-cloud/on-premises/multi-tenancy.md

```diff
@@ -1,40 +1,36 @@
 ---
 id: multi-tenancy
-title: Mult-Tenancy Deployment
+title: Multi-tenancy Deployment
 ---
 
 ##  Overview
-This is intended for system administrators of dbt Cloud to increase the number of accounts 
-supported on their dbt cloud instance. If this is something you are interested in, please contact your
-Account Manager or support (support@getdbt.com). 
+This article is intended for system administrators of dbt Cloud. Follow the steps below to increase the number of accounts supported in your on-premises instance of dbt Cloud. For more information on configuring a multi-tenant deployment of dbt Cloud, contact your Account Manager or support (support@getdbt.com). 
```

Comment:
Leaving notes for myself: The update includes the use case and is clearer on who this should be for (i.e. not hosted).